docs/source/numpy.rst

```diff
@@ -378,3 +391,89 @@ xtensor universal functions are provided for a large set number of mathematical
 +-----------------------------------------------+-----------------------------------------------+
 | ``np.isfinite(a)``                            | ``xt::isfinite(a)``                           |
 +-----------------------------------------------+-----------------------------------------------+
+
+Linear Algebra Functions (xtensor-blas)
+---------------------------------------
+
+Many functions found in the ``numpy.linalg`` module are implemented in `xtensor-blas https://github.com/QuantStack/xtensor-blas`_,
+a seperate package offering BLAS and LAPACK bindings, as well as a convenient interface replicating the ``linalg`` module.
+
+Please note, however, that while we're trying to be as close to NumPy as possible, some features are not 
+implemented yet. Most prominently that is broadcasting for all functions except for ``dot``.
+
+
+**Matrix and vector products**
+
++-----------------------------+--------------------------------------+
+|       Python 3 - numpy      |           C++ 14 - xtensor           |
++=============================+======================================+
+| ``np.dot(a, b)``            | ``xt::linalg::dot(a, b)``            |
++-----------------------------+--------------------------------------+
+| ``np.vdot(a, b)``           | ``xt::linalg::vdot(a, b)``           |
++-----------------------------+--------------------------------------+
+| ``np.outer(a, b)``          | ``xt::linalg::outer(a, b)``          |
++-----------------------------+--------------------------------------+
+| ``np.matrix_power(a, 123)`` | ``xt::linalg::matrix_power(a, 123)`` |
++-----------------------------+--------------------------------------+
+| ``np.kron(a, b)``           | ``xt::linalg::kron(a, b)``           |
++-----------------------------+--------------------------------------+
```

Comment:
Note: the columns must have the same width in rst to have the same width in the css